src/js/grapl-cdk/bin/grapl-cdk.ts

```diff
@@ -5,22 +5,39 @@ import * as cdk from '@aws-cdk/core';
 import { GraplCdkStack } from '../lib/grapl-cdk-stack';
 import { EngagementUx } from '../lib/engagement';
 
-// Deployment parameters
-const deployName = undefined; // ex: 'Grapl-my-deployment' - GRAPL_CDK_DEPLOYMENT_NAME
-const graplVersion = undefined; // defaults to 'latest' - GRAPL_VERSION
-const watchfulEmail = undefined; // (optional) ex: ops@example.com - GRAPL_CDK_WATCHFUL_EMAIL
+// Either replace the `undefined`s or specify the environment variables.
+class DeploymentParameters {
```

Comment:
trying to get the defaults and the `process.env` stuff all in 1 shared namespace